docs/index.rst

```diff
@@ -1,8 +1,76 @@
 ``pluggy``
 ==========
 
-The ``pytest`` plugin system
-****************************
+The five minute elevator pitch
+******************************
+
+``pluggy`` gives `users` the possibility to extend or modify the behaviour of
+a program like e.g. `pytest`_, `tox`_ or `devpi`_ by installing a `plugin` for
+that program. The plugin code will run as part of normal program execution
+changing or enhancing certain aspects of it.
+
+There are three roles in which people might interact with ``pluggy``.
+
+Publisher
```

Comment:
Maybe actually `Host` as in plugin host like described [here](https://en.wikipedia.org/wiki/Plug-in_%28computing%29#Mechanism)?

Reply:
Yes! I caught myself writing host at several points, so this is the right way to go I guess. I will read up a bit more about plugin terminology also ...